pkg/alertmanager/validation/v1alpha1/validation.go

```diff
@@ -300,6 +304,25 @@ func validateTelegramConfigs(configs []monitoringv1alpha1.TelegramConfig) error
 	return nil
 }
 
+func validateWebexConfigs(configs []monitoringv1alpha1.WebexConfig) error {
+	for _, config := range configs {
+		if config.APIURL == "" {
+			if _, err := validation.ValidateURL(config.APIURL); err != nil {
+				return errors.Wrap(err, "invalid 'apiURL'")
+			}
+		}
+
+		if config.RoomID == "" {
+			return fmt.Errorf("mandatory field %q is empty", "roomID")
+		}
```

Comment:
The API definition already says that RoomID is a required field, right? Do we need this validation anyway?

Reply:
Good point; i've removed the unnecessary check :+1: